ui/components/logs/logs-table.tsx

```
@@ -1,159 +1,159 @@
+  }
 
-	const handlePageSizeChange = (newSize: string) => {
-		const newLimit = parseInt(newSize);
-		onPaginationChange({
-			...pagination,
-			limit: newLimit,
-			offset: 0,
-		});
-	};
+  const handlePageSizeChange = (newSize: string) => {
+    const newLimit = parseInt(newSize)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)